src/webfaf/problems.py

```diff
@@ -619,24 +619,24 @@ def bthash_forward(bthash=None):
         return redirect(url_for("problems.item",
                                 problem_id=db_report.problem.id,
                                 component_names=component_names))
-    else:
-        # multiple hashes as get params
-        hashes = request.values.getlist('bth')
-        if hashes:
-            problems = (db.session.query(Problem)
-                        .join(Report)
-                        .join(ReportHash)
-                        .filter(ReportHash.hash.in_(hashes))
-                        .distinct(Problem.id)
-                        .all())
-            if not problems:
-                abort(404)
-            elif len(problems) == 1:
-                return redirect(url_for("problems.item",
-                                        problem_id=problems[0].id,
-                                        component_names=component_names))
-            else:
-                return render_template("problems/multiple_bthashes.html",
-                                       problems=problems)
-        else:
+
+    # multiple hashes as get params
+    hashes = request.values.getlist('bth')
+    if hashes:
+        problems = (db.session.query(Problem)
+                    .join(Report)
+                    .join(ReportHash)
+                    .filter(ReportHash.hash.in_(hashes))
+                    .distinct(Problem.id)
+                    .all())
+        if not problems:
             abort(404)
+        elif len(problems) == 1:
+            return redirect(url_for("problems.item",
+                                    problem_id=problems[0].id,
+                                    component_names=component_names))
+        else:
+            return render_template("problems/multiple_bthashes.html",
+                                   problems=problems)
+    else:
+        abort(404)
```

Comment:
Shouldn't be this part of the code changed as well?

abort is basically just a return.
https://github.com/abrt/faf/blob/d616d941a1d9ccd181a8bdcd90a54ed598659850/src/webfaf/webfaf_main.py#L190